code/modules/hydroponics/grown.dm

```diff
@@ -85,6 +85,18 @@
 			for(var/datum/plant_gene/trait/T in seed.genes)
 				T.on_attackby(src, O, user)
 
+//Ghetto Seed Extraction
+/obj/item/reagent_containers/food/snacks/grown/attackby(obj/item/W, mob/user, params)
+	if(W.get_sharpness())
+		playsound(loc, 'sound/weapons/slice.ogg', 50, TRUE, -1)
+		user.visible_message("<span class='notice'>[user] starts slicing apart \the [src].</span>", "<span class='notice'>You start slicing apart \the [src]...</span>", "<span class='hear'>You hear the sound of a sharp object slicing some plant matter.</span>")
+		if(do_after(user, 50, target = src))
+			to_chat(user, "<span class='notice'>You slice apart this [src].</span>")
```

Comment:
```suggestion
			to_chat(user, "<span class='notice'>You slice apart the [src].</span>")
```
Seems inconsistent with line 92 and reads weirdly in my opinion

Reply:
Yeah I'll change it once I have the other tools in place and all, not quite as it's intended just there so I don't forget what's what